src/Frontend/Components/ProgressBar/ProgressBar.util.tsx

```diff
@@ -37,27 +37,95 @@ export function useOnProgressBarClick(resourceIds: Array<string>) {
   };
 }
 
+const classes = {
+  tooltip: {
+    flex: 1,
+    background: 'rgba(97, 97, 97, 0.92)',
+    borderRadius: '4px',
+    color: 'white',
+    fontFamily: 'Karla Variable, sans-serif',
+    padding: 4,
+    fontSize: 10,
+    margin: 2,
+    fontWeight: 500,
+    position: 'relative',
+    top: 100,
+    'z-index': 100,
+  } as CSSProperties,
+  div: { display: 'flex', 'align-items': 'center' },
+  colorBlock: {
+    display: 'block',
+    width: 12,
+    height: 12,
+    marginRight: 7,
+    marginTop: 2,
+    marginLeft: 2,
+    marginBottom: 2,
+  },
+};
+
 export function getProgressBarTooltipText(
   progressBarData: ProgressBarData,
 ): React.ReactNode {
   return (
-    <MuiBox>
-      Number of resources…
-      <br />
-      …with attributions:{' '}
-      {new Intl.NumberFormat().format(
-        progressBarData.filesWithManualAttributionCount,
-      )}
-      <br />
-      …with only pre-selected attributions:{' '}
-      {new Intl.NumberFormat().format(
-        progressBarData.filesWithOnlyPreSelectedAttributionCount,
-      )}
-      <br />
-      …with only signals:{' '}
-      {new Intl.NumberFormat().format(
-        progressBarData.filesWithOnlyExternalAttributionCount,
-      )}
+    <MuiBox aria-label={'tooltip'} style={classes.tooltip}>
+      Number of resources...
+      <div style={classes.div}>
+        <span
+          style={{
+            ...classes.colorBlock,
+            background: OpossumColors.pastelDarkGreen,
+          }}
+        ></span>
```

Comment:
here and everywhere else:

```suggestion
        <span
          style={{
            ...classes.colorBlock,
            background: OpossumColors.pastelDarkGreen,
          }}
        />
```